sdk/pom.xml

```diff
@@ -42,7 +42,8 @@
     <dependency>
       <groupId>software.amazon.awssdk.crt</groupId>
       <artifactId>aws-crt</artifactId>
-      <version>0.29.11</version>
+      <version>0.29.13</version>
+      <classifier>fips-where-available</classifier>
```

Comment:
why is this necessary?  I thought the default package was what we wanted?

Reply:
crt team encountered building and runtime issues while integrating fips, so they decided to first use a classifier and *maybe* move to default with fips later if there will be no issues.